packages/@aws-cdk/app-delivery/bin/cdk-pipeline

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Can't really do this because it won't work on Windows.

Has to be a JavaScript file.

(And it's not "just a build script", it's in the hot path of every CDK user)

Reply:
Good point about Windows. I’ll convert to JavaScript.